ccmlib/scylla_node.py

```diff
@@ -311,6 +311,7 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
             for node, _ in marks:
                 t = timeout if timeout is not None else 120 if self.cluster.scylla_mode != 'debug' else 360
                 node.watch_rest_for_alive(self, timeout=t)
+                self.watch_rest_for_alive(node, timeout=t)
```

Comment:
We could also call this once with the list of nodes after finishing this loop.

Reply:
but it's not a must

Reply:
Maybe we could make use of ThreadPoolExecutor to wait all the nodes in parallel?

Reply:
I think It's a bit overkill, and it won't make it happen any quicker, i.e. we are waiting for scylla here.